app/policies/organization_page_policy.rb

```diff
@@ -6,15 +6,7 @@ def resolve(action)
     end
   end
 
-  class TranslateScope < Scope
-    def resolve(_action)
-      parent_scope = policy_for(Organization).scope_for(:translate)
-      scope.where(organization_id: parent_scope.select(:id))
-    end
-  end
-
-  scope :index, :show, with: Scope
-  scope :update, :destroy, :update_links, :destroy_links, :versions, :version, with: TranslateScope
+  scope :index, :show, :update, :destroy, :translate, :versions, :version, with: Scope
```

Comment:
I'm not entirely sure about this change. The existing code is saying "you can update a translation if you can translate on the parent". The new code is saying "you can update a translation if you can update the parent" which is not the same.

I know that the translations controller will only ever pass in "translate", but then why is "update" kept in the new list?

And "destroy" was converted to "translate on parent" here in the old code, and thinking about it that was probably wrong (as in we don't want a translator to be able to delete a translation, we only want people with delete access on the parent to be able to delete a translation).